spring/spring5/spring5-remote/src/test/java/org/infinispan/spring/remote/session/InfinispanRemoteSessionRepositoryTest.java

```diff
@@ -37,9 +39,9 @@ public void afterMethod() {
 
    @AfterClass
    public void afterClass() {
+      embeddedCacheManager.stop();
```

Comment:
Why change the order? You normally stop in opposite order to creation. So, if you create embedded cache manager first and then the server, you normally stop server first, then the cache manager.

Reply:
probably changed by mistake. I will put it the way it was. good catch